src/app/components/component-viewer/component-examples/example-viewer/example-viewer.component.html

```diff
@@ -1,10 +1,14 @@
-<hc-tab-set direction="horizontal">
-    <hc-tab [tabTitle]="ext" *ngFor="let ext of ['html', 'ts']">
-        <pre>
-            <hc-doc-viewer [documentUrl]="exampleFileUrl(ext)">
-            </hc-doc-viewer>
-        </pre>
-    </hc-tab>
-</hc-tab-set>
+<div class="hc-tile">
+    <h3 class="docs-api-h3" >Example Name Header</h3>
```

Comment:
Do plan to stick the actual component name here?

Reply:
Ah yes, thanks for calling this out.  I left this in because we talked about having multiple examples on the page.  Breaking up our big pages of examples into smaller chunks so the code is easier to consume.  So these titles at the top of the tile would be the name of the specific example - I just wasn't sure how @jtrujill was planning on dynamically pulling multiple examples in.  But hopefully we can pull a name in as well on those examples that can go in this place.

Reply:
Yeah this information is available for every example. If you look at the examples they have a title annotation and that's how that gets setup

Reply:
Ah perfect, okay I see it now.  Awesome.  Way to think ahead on how you set those up.